homeassistant/components/cast/media_player.py

```diff
@@ -602,7 +602,9 @@ def media_image_url(self):
 
         images = media_status.images
 
-        return images[0].url if images and images[0].url else None
+        return (
+            images[0].url.replace("http://", "//") if images and images[0].url else None
```

Comment:
`images[0].url` is the media image the cast app gave us - if that's using http instead of https that's an issue with the cast app, not HA.

This will probably work in most cases, but if the cast app gives us this URL we should use it IMHO.

Reply:
True. Although when Google's playing a daily briefing or something, it's requesting something from Google's own servers, who don't redirect to HTTPS, or use HSTS.